charts/istio/istio-ingress/templates/priorityclass-istioingressgateway.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: {{ .Values.priorityClassName }}
+value: 1000000000
```

Comment:
This is the priority value that each extension Pod, gardenlet and reversed-vpn-auth-server has. Isn't istio-ingressgateway much more important component actually?

Reply:
This comment is still not addressed.

Reply:
1 billion is the highest available value we can set here. If we want to give it a higher priority in comparison to the other components we have to reduce the priority for them.

Reply:
We could reduce the priority of the other components in a separate PR if you strive for it.

Reply:
> 1 billion is the highest available value we can set here

Okay, I see. I got confused by the doc string in https://github.com/kubernetes/kubernetes/blob/296bf4f01668374ade252a751d4c3567917b9890/staging/src/k8s.io/api/scheduling/v1/types.go#L28-L29 that says that the value can be any int. But the documentation makes it clear.

```
A PriorityClass object can have any 32-bit integer value smaller than or equal to 1 billion. Larger numbers are reserved for critical system Pods that should not normally be preempted or evicted.
```

Ref https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/